hazelcast/src/main/java/com/hazelcast/config/MergePolicyConfig.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+import com.hazelcast.spi.merge.PutIfAbsentMergePolicy;
+
+import java.io.IOException;
+
+import static com.hazelcast.util.Preconditions.checkHasText;
+import static com.hazelcast.util.Preconditions.checkPositive;
+
+/**
+ * Configuration for {@link com.hazelcast.spi.SplitBrainMergePolicy}.
+ */
+public class MergePolicyConfig implements IdentifiedDataSerializable {
```

Comment:
Can you add some javadoc on the getters and setters?

Reply:
Good catch, done!